### PR TITLE
Changes to avoid numerical inconveniences

### PR DESCRIPTION
**Pull request by AFranci**

### CODE REVIEW

**Comment by AFranci** [comment]
some feedback?

File: applications/swimming_DEM_application/custom_utilities/binbased_DEM_fluid_coupled_mapping.cpp
Comment (maceligueta) [inline applications/swimming_DEM_application/custom_utilities/binbased_DEM_fluid_coupled_mapping.cpp:1228]: missing & ? otherwise we are not modifying `i_node->FastGetSolutionStepValue(NODAL_AREA)`

Comment (maceligueta) [reply applications/swimming_DEM_application/custom_utilities/binbased_DEM_fluid_coupled_mapping.cpp:1228]: nothing

**Review by maceligueta — APPROVED** [review]

**Comment by maceligueta** [comment]
But I think we should wait for @GuillermoCasas approval.

**Comment by GuillermoCasas** [comment]
Ok!

**Pull request merged**
